graders/r-previous/README.md

```diff
@@ -1,4 +1,4 @@
-## PrairieLearn Autograding R Docker Image
+# PrairieLearn Autograding R Docker Image
```

Comment:
The other grader images use the image name as the heading. Can we do the same here and in `graders/r`?

Reply:
We aren't building `r-previous`. Should we remove from the repo?

Reply:
I think that's reasonable to do, yeah.

Reply:
Split in #11713

Reply:
I also fixed up `graders/r` in 33b8c2062e43169263306f31af780d819f3f3c93